src/modules/magento-2-cms/components/CmsData.vue

```diff
@@ -54,7 +54,7 @@ export default {
       return currentStoreView()
     },
     storeView () {
-      return (this.isMultistoreEnable && this.currentStore) ? this.currentStore.storeId : 0
+      return (this.isMultistoreEnable && this.currentStore) ? this.currentStore.storeId : 1
```

Comment:
since storeID is set to '1' by default it can be false therefore the last check is redundant. Am I right?

Reply:
@filrak I don't think so.
what the above condition seems to do is:
- if you have multistore and the current store has been correctly set, use `this.currentStore.storeId` (from multistore module)
- else (multistore is disabled or this.currentStore is empty) use cms fallback (not taken from multistore fallback)
there might be some magic logic I cannot find but when we tried, both of those had to be set separately (depending on multistore setup we had). Based on that I think this still require to set BOTH fallbacks this way.